pkg/option/config.go

```diff
@@ -1017,6 +1017,10 @@ const (
 
 	// VTEP MACs
 	VtepMAC = "vtep-mac"
+
+	// TCFilterPriority sets the priority of the cilium tc filter, enabling other
+	// filters to be inserted prior to the cilium filter.
+	TCFilterPriority = "tc-filter-priority"
```

Comment:
Small nit, but otherwise the rest lgtm. Could you name this `bpf-filter-priority` ? Most low-level opts in the agent have a `bpf` prefix, would be good to keep it consistent.

Reply:
Thanks!